docs/intro/overrides.rst

```diff
@@ -0,0 +1,349 @@
+        def to_item(self):
+            ...  # more specific parsing
+
+
+    @handle_urls("anotherexample.com", overrides=GenericProductPage, exclude="/digital-goods/")
+    class AnotherExampleProductPage(ItemWebPage):
+        def to_item(self):
+            ...  # more specific parsing
+
+
+    @handle_urls(["dualexample.com/shop/?product=*", "dualexample.net/store/?pid=*"], overrides=GenericProductPage)
+    class DualExampleProductPage(ItemWebPage):
+        def to_item(self):
+            ...  # more specific parsing
+
+The code above declares that:
+
+    - For sites that match the ``example.com`` pattern, ``ExampleProductPage``
+      would be used instead of ``GenericProductPage``.
+    - The same is true for ``DualExampleProductPage`` where it is used
+      instead of ``GenericProductPage`` for two URL patterns which works as
+      something like:
+
+      - :sub:`(match) https://www.dualexample.com/shop/electronics/?product=123`
+      - :sub:`(match) https://www.dualexample.com/shop/books/paperback/?product=849`
+      - :sub:`(NO match) https://www.dualexample.com/on-sale/books/?product=923`
+      - :sub:`(match) https://www.dualexample.net/store/kitchen/?pid=776`
+      - :sub:`(match) https://www.dualexample.net/store/?pid=892`
+      - :sub:`(NO match) https://www.dualexample.net/new-offers/fitness/?pid=892`
+
+    - On the other hand, ``AnotherExampleProductPage`` is only used instead of
+      ``GenericProductPage`` when we're handling pages from ``anotherexample.com``
+      that doesn't contain ``/digital-goods/`` in its URL path.
+
+.. tip::
+
+    The URL patterns declared in the ``@handle_urls`` annotation can still be
+    further customized. You can read some of the specific parameters in the
+    :ref:`API section <api-overrides>` of :func:`web_poet.handle_urls`.
+
+
+Retrieving all available Overrides
+----------------------------------
+
+The :meth:`~.PageObjectRegistry.get_overrides` method from the ``web_poet.default_registry``
+allows discovery and retrieval of  all :class:`~.OverrideRule` from your project.
+Following from our example above, using it would be:
+
+.. code-block:: python
+
+    from web_poet import default_registry
+
+    # Retrieves all OverrideRules that were registered in the registry
+    rules = default_registry.get_overrides()
+
+    print(len(rules))  # 3
+    print(rules[0])    # OverrideRule(for_patterns=Patterns(include=['example.com'], exclude=[], priority=500), use=<class 'my_project.page_objects.ExampleProductPage'>, instead_of=<class 'my_project.page_objects.GenericProductPage'>, meta={})
+
+Remember that using ``@handle_urls`` to annotate the Page Objects would result
+in the :class:`~.OverrideRule` to be written into ``web_poet.default_registry``.
+
+
+.. warning::
+
+    :meth:`~.PageObjectRegistry.get_overrides` relies on the fact that all essential
+    packages/modules which contains the :func:`web_poet.handle_urls`
+    annotations are properly loaded.
```

Comment:
```suggestion
    annotations are properly loaded (i.e. imported).
```

Reply:
Overall, I think it may be a bit more clear if in some places further in the docs instead of "loaded" is replaced with "imported". consume_modules is not doing anything fancy, it just imports modules.

Reply:
Updated this on https://github.com/scrapinghub/web-poet/pull/27/commits/37e5ba1a3696f629801478fe78ea1729164f39df